File: overview_new/src/main/java/io/innofang/overview_new/BannerViewHolder.java
package io.innofang.overview_new;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.widget.ImageView;

import com.zhouwei.mzbanner.holder.MZViewHolder;

/**
 * Author: Inno Fang
 * Time: 2017/8/19 12:37
 * Description:
 */


public class BannerViewHolder implements MZViewHolder<Integer> {
    private ImageView mImageView;
    @Override
    public View createView(Context context) {
        // 返回页面布局
        View view = LayoutInflater.from(context).inflate(R.layout.ov_layout_banner_item, null);
        mImageView = (ImageView) view.findViewById(R.id.banner_image_view);
        return view;
    }

    @Override
    public void onBind(Context context, int position, Integer data) {
        // 数据绑定
        mImageView.setImageResource(data);
    }
}

File: overview/src/main/java/cn/edu/nuc/overview/BannerViewHolder.kt
package cn.edu.nuc.overview

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.widget.ImageView
import com.zhouwei.mzbanner.holder.MZViewHolder

/**
 * Author: Inno Fang
 * Time: 2017/8/18 13:15
 * Description:
 */

class BannerViewHolder:MZViewHolder<Int> {

    private lateinit var imageView: ImageView

    override fun createView(context: Context?): View {
        val view = LayoutInflater.from(context)
                .inflate(R.layout.ov_layout_banner_item, null)
        imageView = view.findViewById(R.id.banner_image_view) as ImageView
        return view
    }

    override fun onBind(context: Context?, pos: Int, data: Int?) {
       imageView.setImageResource(data!!)
    }


}